microprofile/tests/arquillian/src/main/java/io/helidon/microprofile/arquillian/HelidonMethodExecutor.java

```diff
@@ -83,6 +88,26 @@ public TestResult invoke(TestMethodExecutor testMethodExecutor) {
         return TestResult.passed();
     }
 
+    /**
+     * Inject URL in a hard way.
+     *
+     * @param testMethodExecutor
+     * @throws java.net.MalformedURLException
+     * @throws IllegalAccessException
```

Comment:
missing descriptions on when these exceptions are thrown, as well as description of the parameter

Reply:
Done